docs/utilities/tags.md

```diff
@@ -35,25 +35,49 @@ Using Tags In Code
 ------------------
 Tags for all registries are automatically sent from the server to any remote clients on login and reload. `Block`s, `Item`s, `EntityType`s, `Fluid`s, and `GameEvent`s are special cased as they have `Holder`s allowing for available tags to be accessible through the object itself.
```

Comment:
To my knowledge, the instrusive `Holder.Reference` held by instances of the foregoing are likely to be removed in some future version of Minecraft. This may be worth noting so modders query the registry instead, therefore saving themselves the hassle when the removal is done and they port their mod forward.